src/js/components/TimeAgo.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import DateUtil from '../utils/DateUtil';
+import Util from '../utils/Util';
+
+const METHODS_TO_BIND = [
+  'updateTime'
+];
+
+class TimeAgo extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    });
+
+    this.state = {
+      interval: global.setInterval(this.updateTime, this.props.updateInterval)
```

Comment:
I think this needs to change dynamically based on the time diff. It won't make sense to change every 30 seconds when something happened hours or days ago. Or even minutes. It would be nice if the component was smart enough to change this dynamically!